pyServer/GuestFS.py

```diff
@@ -232,6 +234,14 @@ def case_sensitive_path(self, path):
         (out, err) = self.callGF('Finding Case Sensitive Path [' + path + ']', ['--', '-case-sensitive-path', path], True)
         return self.get_first_list_item(out)
         
+    def is_symlink(self, path):
```

Comment:
is this `is_symlink()` operation actually used?

Reply:
Added as apart of another task. But removing it from this PR